build.py

```diff
@@ -386,7 +386,10 @@ def build_package(target, build_native_package):
         package_cmd.extend(['-t', get_package_type()])
         package_cmd.extend(['-n', 'irods-externals-{0}'.format(package_subdirectory)])
         try:
-            if get_package_type() == 'rpm' and v['rpm_dependencies'] and 'Centos' in platform.linux_distribution()[0].capitalize():
+            # TODO AlmaLinux 8: Is this change necessary?
+            # Mungefs appears to be the only package that experienced issues.
+            #if get_package_type() == 'rpm' and v['rpm_dependencies'] and any(x in platform.linux_distribution()[0].capitalize() for x in ['Centos', 'AlmaLinux']):
+            if get_package_type() == 'rpm' and v['rpm_dependencies'] and any(x in platform.linux_distribution()[0].capitalize() for x in ['Centos']):
                 for d in v['rpm_dependencies']:
                     package_cmd.extend(['-d', d])
```

Comment:
I'm not sure about L391. Externals build fine against AlmaLinux. Mungefs fails, but that is because the Mungefs CMakeLists.txt file needs a small tweak.

Reply:
this could be a leftover... not sure.  let's leave this one unresolved for now...

Reply:
There are two references to `rpm_dependencies` in versions.json (one for nanodbc and the other for libarchive).

For nanodbc, the installation of unixODBC-devel brings in unixODBC.

I don't know what is allowing libarchive to build. Seems xz-libs is possibly being installed by some other dependency as well.

I guess the only real way to determine if AlmaLinux should trigger L393 is to try and compile/run iRODS on AlmaLinux.

Reply:
Added AlmaLinux because this code is used to declare deps for the package produced by this repo.